google-cloud-firestore/src/main/java/com/google/cloud/firestore/ResourcePath.java

```diff
@@ -98,26 +98,46 @@ String getId() {
   }
 
   /**
-   * String representation as expected by the Firestore API.
+   * The Path's name (the location relative to the root of the database).
    *
-   * @return The formatted name of the resource.
+   * @return The resource path relative to the root of the database.
    */
-  @Override
-  public String toString() {
+  String getPath() {
     StringBuilder result = new StringBuilder();
-    result.append(getDatabaseName());
 
-    ImmutableList<String> parts = getSegments();
-    if (parts.size() > 0) {
-      result.append("/documents");
-      for (String part : parts) {
+    boolean first = true;
```

Comment:
Isn't this `String.join("/", getSegments())`?

Reply:
It is, but using the Java 7 API. Did we move to Java 8?

Reply:
Oh duh, sorry.